config/webpack.config.prod.js

```diff
@@ -76,28 +81,31 @@ module.exports = {
   // directory of `react-scripts` itself rather than the project directory.
   // You can remove this after ejecting.
   resolveLoader: {
-    root: paths.ownNodeModules,
-    moduleTemplates: ['*-loader'],
+    // TODO (hstastna): what to do with this? it does not work for webpack > 1
+    // see also https://github.com/webpack/webpack/issues/3066
+    // root: paths.ownNodeModules,
+    // moduleTemplates: ['*-loader'],
   },
   module: {
-    // First, run the linter.
-    // It's important to do this before Babel processes the JS.
-    preLoaders: [
+    // For webpack >= 2, module.loaders changed to module.rules
+    rules: [
       {
+        // module.preLoaders were removed
+        // First, run the linter.
+        // It's important to do this before Babel processes the JS.
         test: /\.(js|jsx)$/,
-        loader: 'eslint',
+        loader: 'eslint-loader',
         include: paths.appSrc,
         options: {
           failOnError: true,
-        }
+        },
```

Comment:
I think you can add `configFile: path.join(__dirname, 'eslint.js'),` here and be ok and not need it later in the `LoaderOptiosnsPlugin`